lib/routes/home/payments_filter.dart

```diff
@@ -140,23 +143,26 @@ class PaymentsFilterState extends State<PaymentsFilter> {
 
   Theme _buildFilterDropdown(BuildContext context) {
     return Theme(
-      data: theme.themeId == "BLUE"
-          ? Theme.of(context)
-          : Theme.of(context)
-              .copyWith(canvasColor: Theme.of(context).backgroundColor),
+      data: Theme.of(context).copyWith(
+          canvasColor: theme.themeId == "BLUE"
+              ? Color.fromRGBO(249, 249, 249, 1)
```

Comment:
Can we have all these constants in the theme_data ?

Reply:
[Done](https://github.com/breez/breezmobile/pull/370/commits/b553a2483b185ef03b56b0b326721c5ee9125c2b)